### ISSUE TITLE
自己的数据，跑出来效果不好

### ISSUE THREAD

**Issue by zhanghaoxuan1999** [649453932/Bert-Chinese-Text-Classification-Pytorch#117]

**Comment by zhanghaoxuan1999** [comment]
参数：
        self.num_epochs = 20                                             # epoch数
        self.batch_size = 8                                           # mini-batch大小
        self.pad_size = 128                                              # 每句话处理成的长度(短填长切)
        self.learning_rate = 1e-5                                       # 学习率
        self.bert_path = './bert_pretrain'
        self.tokenizer = BertTokenizer.from_pretrained(self.bert_path)
        self.hidden_size = 768

**Comment by zhanghaoxuan1999** [comment]
Test Loss:  0.58,  Test Acc: 63.33%
Precision, Recall and F1-Score...
              precision    recall  f1-score   support

          目的     0.9083    0.9706    0.9384       102
          方法     0.4200    0.2258    0.2937        93
          结论     0.4965    0.6667    0.5691       105

    accuracy                         0.6333       300
   macro avg     0.6082    0.6210    0.6004       300
weighted avg     0.6128    0.6333    0.6093       300

**Comment by zhanghaoxuan1999** [comment]
Test Loss:  0.58,  Test Acc: 63.33%
Precision, Recall and F1-Score...
              precision    recall  f1-score   support

          目的     0.9083    0.9706    0.9384       102
          方法     0.4200    0.2258    0.2937        93
          结论     0.4965    0.6667    0.5691       105

    accuracy                         0.6333       300
   macro avg     0.6082    0.6210    0.6004       300
weighted avg     0.6128    0.6333    0.6093       300

Confusion Matrix...
[[99  0  3]
 [ 4 21 68]
 [ 6 29 70]]
Time usage: 0:00:01

**Comment by zhanghaoxuan1999** [comment]
已解决，数据量不够，增加数据量，结果不错。

**Comment by 7Z-learner** [comment]
你好，请问跑自己的数据集的话是用‘python run.py --model xxx --embedding random --word True’吗？应该不用预先跑其他代码生成预训练的词向量吧？感谢！

**Comment by zhanghaoxuan1999** [comment]
> 

不需要，博主已经写好了，只需要填充我们的数据就可以。

**Comment by zhanghaoxuan1999** [comment]
> 你好，请问跑自己的数据集的话是用‘python run.py --model xxx --embedding random --word True’吗？应该不用预先跑其他代码生成预训练的词向量吧？感谢！

不需要弄别的（如果没有特殊需求），直接替换数据就可以

**Comment by MewemeW** [comment]
请问加了多少数据量，结果就变得不错了呢？

**Comment by asdjklfgh** [comment]
直接用的附带的数据集，但是跑出来的效果很差，请问您用了多大的数据量跑出来效果不错的呢？
